src/Functions/greatCircleDistance.cpp

```diff
@@ -43,48 +43,43 @@ namespace
 {
 
 constexpr double PI = std::numbers::pi_v<double>;
-constexpr float PI_F = std::numbers::pi_v<float>;
+constexpr double PI_F = std::numbers::pi_v<double>;
```

Comment:
`PI_F` is supposed to mean PI float, so it would make more sense to just delete this and similar lines and replace all `PI_F` with `PI`

Reply:
Gotcha, done.